vendor-bin/doctrine_phpcr/composer.json

```diff
@@ -0,0 +1,14 @@
+{
+    "require-dev": {
+        "doctrine/data-fixtures": "^1.2",
+        "doctrine/dbal": "^2.5",
+        "doctrine/phpcr-odm": "^1.4",
+        "jackalope/jackalope-doctrine-dbal": "^1.2",
+        "symfony/console": "^2.5",
```

Comment:
Is required to run the testsuite as the `cli-config.php` used to initialize PHPCR db schema requires console `HelperSet` class to be available.